tests/capture-tests.js

```diff
@@ -243,26 +265,46 @@ require(["mobifyjs/utils", "capture"], function(Utils, Capture) {
         $("#qunit-fixture").append($iframe);
     });
 
+    // Ensure that a meta viewport tag is added to the document if one is not present
```

Comment:
Yep that comment is out of date. `ios8AndGreatScrollFix` will always inject a meta tag into the provided document. Fixed: https://github.com/mobify/capturejs/commit/78b8e254faeba0421db833986322d612baac78a5